website/docs/reference/project-default-strategy.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Project default strategy
+---
+
```

Comment:
I suggest using figures for some of the images. To do that, we need to import the component.

```suggestion
import Figure from '@site/src/components/Figure/Figure.tsx'

```